packages/showcase/src/app/utility-classes/spacing/spacing.component.ts

```diff
@@ -1,11 +1,105 @@
-import { ChangeDetectionStrategy, Component } from "@angular/core";
+import {
+  ChangeDetectionStrategy,
+  Component,
+  computed,
+  CUSTOM_ELEMENTS_SCHEMA,
+  inject,
+  input
+} from "@angular/core";
+import { CommonModule } from "@angular/common";
+import { Router } from "@angular/router";
+
+import type { ChCheckboxCustomEvent } from "@genexus/chameleon-controls-library";
+
+import { spacingMetadata } from "./metadata";
+import { CSS } from "../../../common/html";
+import { CodeSnippetComponent } from "../../../user-controls/code-snippet/code-snippet.component";
 import { RuntimeBundlesComponent } from "../../../user-controls/runtime-bundles/runtime-bundles.component";
+import { CommonSnippets } from "../../../user-controls/common-snippets/common-snippets.component";
 
 @Component({
-  selector: "utility-classes-spacing",
+  selector: "components-spacing",
```

Comment:
This component is under the `utility-classes` folder. Also, the "components" prefix is only used for the templates that are under the `components` folder.

```suggestion
  selector: "utility-classes-spacing",
```